src/priority_q.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+# -*- coding: utf -8 -*-
+
+#import pdb; pdb.set_trace()
+
+from __future__ import unicode_literals, division
+
+from binary_heap import BinaryHeap
+
+
+class Item(object):
```

Comment:
Yall are over-complicating this a bit. There already exists a built in Python type with greater than/less than comparison methods which is a great fit to store values and their priorities in the heap.